norm failed. needs more precision constraints?

```
FAIL: test_NormFunction_1_5_dim (__main__.TestAutograd)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_autograd.py", line 2065, in do_test
    self.assertTrue(gradgradcheck(apply_fn, input, grad_y,))
AssertionError: False is not true
======================================================================
FAIL: test_NormFunction_1_5_dim_neg0 (__main__.TestAutograd)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_autograd.py", line 2065, in do_test
    self.assertTrue(gradgradcheck(apply_fn, input, grad_y,))
AssertionError: False is not true

```

--- a/test/test_autograd.py
+++ b/test/test_autograd.py
@@ -2029,7 +2029,7 @@ for test in function_tests:
 
         def do_test(self, cls=cls, constructor_args=new_constructor_args,
                     call_args=new_call_args, test_name=test_name):
-            input = create_input(call_args, non_contiguous=True if not "View" == cls.__name__ else False)
+            input = create_input(call_args, non_contiguous="View" not in cls.__name__)
             if cls._is_legacy:
                 def apply_fn(*input):
                     return cls(*constructor_args)(*input)

